pyproject.toml

```diff
@@ -76,19 +63,25 @@ known_first_party = [
 [tool.mypy]
 
 [[tool.mypy.overrides]]
+# Note: Only globally ignore missing imports as a matter of last resort!
+# See https://blog.wolt.com/engineering/2021/09/30/professional-grade-mypy-configuration/
 module = [
     # Add external modules w/o type hints here
 ]
 ignore_missing_imports = true
 
 [tool.flake8]
 max-line-length = 88
+# max-complexity = 10
```

Comment:
I would strongly encourage us to include some kind of complexity metric. Just to raise awareness for it and to make our code as readable as possible.

Reply:
I thought this would only change the default, but just did some checking and googling, and turns out this must be set for the check to run ("By default the plugin is disabled. Use the --max-complexity switch to enable it.", see [here](By default the plugin is disabled. Use the --max-complexity switch to enable it.)).